src/ml_board/frontend/Dashboard/src/components/topbar-with-drawer/TopBarWithDrawer.tsx

```diff
@@ -0,0 +1,197 @@
+                            <ListItem key={index} disablePadding onClick={() => changeTabRequest(RoutesMapping[routeMapKey].url)}>
+                                <ListItemButton 
+                                    selected={
+                                        RoutesMapping[routeMapKey].url === currentTab ? 
+                                        true 
+                                        :
+                                        false
+                                    }
+                                    >
+                                    <ListItemIcon>
+                                        {RoutesMapping[routeMapKey].menuIcon}
+                                    </ListItemIcon>
+                                    <ListItemText primary={routeMapKey} />
+                                </ListItemButton>
+                            </ListItem>
+                        )
+                    }
+                    else {
+                        return null
+                    }
+                })
+            }
+        </List>
+        <Divider />
+        <Throughput/>
+    </Box>
+    );
+
+  return (
+    <Box sx={{ display: 'flex' }}>
+        {/* Title Bar */}
+        <AppBar 
+            component="nav" 
+            sx={{ 
+                backgroundColor: "#FFFFFF",
+                flexGrow: 1
+            }}
+        >
+            <Toolbar>
+                {/* Hamburger Menu Icon */}
+                <IconButton
+                    size="large"
+                    edge="start"
+                    aria-label="open drawer"
+                    sx={{ 
+                        mr: 2,
+                        borderRadius: "10px",
+                        border: "1px solid",
+                        borderColor: "#E0E3E7"
+                    }}
+                    onClick={toggleDrawer("left", true)}
+                >
+                    <MenuIcon />
+                </IconButton>
+                
+                {/* Page Title Text Goes Here */}
+                <Container
+                    sx={{
+                        display: {display: "flex"},
+                        direction: "row",
+                        alignItems:"center",
+                        justifyContent: "center"
+                    }}
+                >
+                    <Typography
+                        component="div"
+                        variant="h6"
+                        sx={{
+                            display: {color: "black", fontWeight: "bold", fontSize: 30}
+                        }}
+                    >
+                        {
+                            location.pathname.split("/")[1] === "" ?
+                            RoutesMapping.Graphs.url.charAt(0).toUpperCase() + RoutesMapping.Graphs.url.slice(1)
+                            :
+                            location.pathname.split("/")[1].charAt(0).toUpperCase() + location.pathname.split("/")[1].slice(1)
```

Comment:
Just capitalizing the first letter of the selected page name. As the name from `location.pathname.split("/")[1]` would be the same as RoutesMapping.<MenuKey>.url --> it will be in lowercase.
And, if the URL === " " then it's the home page, which will be the Graphs page by default.